### PR TITLE
Notification App - Extract strings for translation

### PR DESCRIPTION
**Pull request by TerenceLiew**
:haircut: 
:koala: 
/cc @zendesk/quokka 

### Description
As per title. Probably not standard, looking for improvements.
### References
JIRA Link - https://zendesk.atlassian.net/browse/OFFAPPS-417

### Risks
None - Low ? If translations are missing or referring to different hash path

### CODE REVIEW

**Comment by princemaple** [comment]
take a look at https://github.com/zendesk/answer_suggestion_app/blob/master/translations/en.json
I'd like to see all the translations in the `{title,value}` pair format.
Title is for translators, so it basically describe the context of the value.

When everything is done, run `zat translate to_yml` to generate `en.yml`, the translators will then use en.yml to do translations.

**Comment by TerenceLiew** [comment]
Ah yea totally forgot about the yml file sorry!

**Comment by princemaple** [comment]
That's everything? Cool, :+1:

**Comment by TerenceLiew** [comment]
Thanks again @princemaple !

**Comment by sandlerr** [comment]
@TerenceLiew looking at other apps do they use camelCase in their string keys?

**Comment by sandlerr** [comment]
...yes they do
:+1:

**Comment by TerenceLiew** [comment]
Sorry @sandlerr I disappeared for a moment and only just got back

File: translations/en.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      title: "The button to click on cancel"
+      value: "Cancel"
+  - translation:
+      key: "txt.apps.notification_app.button.imageInput"
+      title: "The button to allow users to add an image link"
+      value: "Add image link"
+  - translation:
+      key: "txt.apps.notification_app.button.sendMessage"
+      title: "The button to broadcast a message"
+      value: "Broadcast"
```
Comment (ydrain) [inline translations/en.yml]: We need more information about "Broadcast". This term doesn't work in many languages in this case, and we need to give instruction how to translate. Please describe what "Broadcast" does and add the url of screenshot.

Comment (princemaple) [inline translations/en.yml]: @ydrain It's a button that once you click, your message will be send to all agents. This is used by an admin.

Comment (ydrain) [inline translations/en.yml]: Please add the information & this url (https://zendesk.box.com/s/1bs135f0i9mo47won2m5h9igxo514kan) in the 'title'. The translators only see the title & value fields nothing else. Thanks!

**Pull request merged**